src/types/index.ts

```diff
@@ -340,7 +348,7 @@ export type UserTransactionResponse = {
   hash: string;
   state_change_hash: string;
   event_root_hash: string;
-  state_checkpoint_hash?: string;
+  state_checkpoint_hash: string | null;
```

Comment:
The `?` is specifically for undefined?  Is there a reason to switch to `string | null`?

Reply:
yes.  the API returns null, not undefined.  So the attribute exists (is defined), but is set to null.

If you didn't make this change the test would fail (compilation)